assets/styles/common/index.scss

```diff
@@ -16,3 +16,4 @@
 @import 'survey.scss';
 @import 'user.scss';
 @import 'utils.scss';
+@import 'list.scss';
```

Comment:
I wasn't 100% clear on the scss structure and where this should go. Hopefully this was reasonable.